File: 백준/Bronze/2588. 곱셈/곱셈.java
import java.util.Scanner;

class Main {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int A = sc.nextInt();
		int B = sc.nextInt();
		sc.close();
		int B1 = B % 10;
		int B10 = (B / 10) % 10;
		int B100 = (B / 100) % 10;

		System.out.println(A * B1);
		System.out.println(A * B10);
		System.out.println(A * B100);
		System.out.println(A * B);
	}
}

File: 백준/Bronze/2588. 곱셈/곱셈.py
A = int(input())
B = int(input())

B1 = int(B % 10)
B10 = int((B / 10) % 10)
B100 = int((B / 100) % 10)

print(A * B1)
print(A * B10)
print(A * B100)
print(A * B)